_includes/integrations/react-tech-ref.md

```diff
@@ -13,51 +13,104 @@
   - [`initialValue`](#initialvalue)
   - [`inline`](#inline)
   - [`onEditorChange`](#oneditorchange)
-  - [`outputFormat`](#outputformat)
+  - [`outputFormat` (*deprecated*)](#outputformat)
   - [`plugins`](#plugins)
+  - [`scriptLoading`](#scriptloading)
   - [`tagName`](#tagname)
   - [`textareaName`](#textareaname)
   - [`toolbar`](#toolbar)
   - [`tinymceScriptSrc`](#tinymcescriptsrc)
   - [`value`](#value)
 
+- [Using the TinyMCE React component as a uncontrolled component](#usingthetinymcereactcomponentasauncontrolledcomponent)
 - [Using the TinyMCE React component as a controlled component](#usingthetinymcereactcomponentasacontrolledcomponent)
 - [Event binding](#eventbinding)
 
-### Installing the TinyMCE React integration using NPM
+### Installing the TinyMCE React integration using NPM or Yarn
 
 To install the `tinymce-react` package and save it to your `package.json`.
 
 ```sh
 $ npm install --save @tinymce/tinymce-react
 ```
 
-### Configuring the editor
-
-The editor accepts the following properties:
+or with Yarn
 
-```jsx
-<Editor
-  apiKey='your-api-key'
-  cloudChannel='{{site.productmajorversion}}-stable'
-  disabled={false}
-  id='uuid'
-  init= {% raw %}{{{% endraw %}  {% raw %}}}{% endraw %}
-  initialValue=''
-  inline={false}
-  onEditorChange={}
-  plugins=''
-  tagName='div'
-  textareaName=''
-  toolbar=''
-  value=''
-/>
+```sh
+$ yarn add @tinymce/tinymce-react
 ```
 
-None of the configuration properties are **required** for `tinymce-react` to work. Specify a {{site.cloudname}} API key using `apiKey` to remove the `This domain is not registered...` warning message.
+### Configuring the editor
+
+The following are the most common properties grouped by category.
+
+#### Configuring editor source
```

Comment:
```suggestion
#### Configuring editor source

```
Adding a new line for style consistency.

Reply:
done